### PR TITLE
St/various improvements

### PR DESCRIPTION
**Pull request by shanebo**
@scottcorgan core changes:

1. This renames the library
2. Adds you as a contributor to package.json
3. Refactors the variable names and methods and utilizes some es6 APIs
4. Adds a listen and unlisten function, for teardown and resetting say after an ajax request

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -100,32 +82,36 @@ module.exports = function suiter(elementSelector) {
...
     });
 
     resizing = false;
   }
 
-  window.addEventListener('resize', throttle);
-  resize();
+  return {
+    listen: function() {
+      resizing = false;
```
Comment (scottcorgan) [inline index.js:105]: 👍 This function looks much cleaner!

File: package.json [hunk 2]
```diff
@@ -1,20 +1,25 @@
 {
-  "name": "suiter",
-  "version": "1.0.0",
+  "name": "element-breakpoints",
+  "version": "0.5.0",
```
Comment (scottcorgan) [inline package.json]: Not a huge deal, but it's recommended by npm to start all package at `1.0.0` if following semver.

**Review by scottcorgan — APPROVED** [review]
On small note.

But, good stuff man. As long as the tests pass, there's no blockers!

File: package.json (hunk 2 above)
Comment (shanebo) [reply package.json]: I'll change back to `1.0.0` then. Thanks.

**Pull request merged**